src/libraries/pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.builds

```diff
@@ -1,7 +1,8 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
-  <ItemGroup>
+  <!-- only build during the AllConfigurations leg -->
+  <ItemGroup Condition="'$(BuildAllConfigurations)' == 'true'">
```

Comment:
Is this package desired during local builds? Should it be more similar to how the targets package works?

https://github.com/dotnet/runtime/blob/6240a608671a82016a877f1cf7e0265c21cae241/src/libraries/pkg/Microsoft.NETCore.Targets/Microsoft.NETCore.Targets.builds#L4-L6

Reply:
cc @ericstj

Reply:
Just AllConfigurations is ok.  In fact, we might be able to delete this package completely.  It was previously here because we were building WCF at the same time as corefx and they needed to use our packaging system along with our index file to calculate dependencies in the same way.

Now WCF is using the regular .NET SDK for packaging, so I think we can delete this package completely! (and move the index to some more sensible location.  Let me double check consumers.

Reply:
Looks safe to remove, I don't see any downstream consumers.

Reply:
In order to unblock the official build, let's let the CI run to completion and merge this fix before working on removal. (Probably stating the obvious, but want to make sure.)

Reply:
sry @dagood I saw the comment after pushing the commit.